include/inexor/vulkan-renderer/application.hpp

```diff
@@ -26,31 +26,38 @@ class Application : public VulkanRenderer {
     std::vector<std::string> m_shader_files;
     std::vector<std::string> m_gltf_model_files;
 
-    /// @brief Loads the configuration of the renderer from a TOML configuration file.
-    /// @brief file_name [in] The TOML configuration file.
-    /// @note It was collectively decided not to use JSON for configuration files.
+    /// @brief
+    /// @param file_name
     void load_toml_configuration_file(const std::string &file_name);
 
+    /// @brief
     VkResult load_textures();
```

Comment:
I'm not done yet here but I wanted to push just already. Will finish and squash later.